server/api/user/user.controller.js

```diff
@@ -5,6 +5,9 @@ import passport from 'passport';
 import config from '../../config/environment';
 import jwt from 'jsonwebtoken';
 import Section from "../section/section.model";
+import Sendgrid from '../../components/email';
+import verificationTokenModel from '../../auth/local/verificationToken.model';
```

Comment:
Not consistent styling on "verificationTokenModel" compared to "User" and "Section"